{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"failureReason"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}],"languages":["swift"],"platforms":["macOS"]}]}],"schemaVersion":{"major":0,"minor":2,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/authfoundation\/keychainerror\/failurereason"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation\/KeychainError\/failureReason","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"A localized message describing the reason for the failure."}],"kind":"symbol","metadata":{"role":"symbol","title":"failureReason","roleHeading":"Instance Property","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"failureReason"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"symbolKind":"property","externalID":"s:10Foundation14LocalizedErrorPAAE13failureReasonSSSgvp::SYNTHESIZED::s:14AuthFoundation13KeychainErrorO","extendedModule":"Foundation","modules":[{"name":"AuthFoundation"}]},"hierarchy":{"paths":[["doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation","doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation\/KeychainError","doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation\/KeychainError\/LocalizedError-Implementations"]]},"references":{"doc://com.okta.mobile.authfoundation/documentation/AuthFoundation":{"role":"collection","title":"AuthFoundation","abstract":[{"type":"text","text":"Library that provides foundational features used by Oktaâ€™s authentication libraries, as well as APIs used to work with tokens and user credentials."}],"identifier":"doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation","kind":"symbol","type":"topic","url":"\/documentation\/authfoundation"},"doc://com.okta.mobile.authfoundation/documentation/AuthFoundation/KeychainError/LocalizedError-Implementations":{"role":"collectionGroup","title":"LocalizedError Implementations","abstract":[],"identifier":"doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation\/KeychainError\/LocalizedError-Implementations","kind":"article","type":"topic","url":"\/documentation\/authfoundation\/keychainerror\/localizederror-implementations"},"doc://com.okta.mobile.authfoundation/documentation/AuthFoundation/KeychainError":{"role":"symbol","title":"KeychainError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"KeychainError"}],"abstract":[{"type":"text","text":"Describes errors that may occur when interacting with the keychain."}],"identifier":"doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation\/KeychainError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"KeychainError"}],"url":"\/documentation\/authfoundation\/keychainerror"},"doc://com.okta.mobile.authfoundation/documentation/AuthFoundation/KeychainError/failureReason":{"role":"symbol","title":"failureReason","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"failureReason"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"kind":"text","text":"?"}],"abstract":[{"type":"text","text":"A localized message describing the reason for the failure."}],"identifier":"doc:\/\/com.okta.mobile.authfoundation\/documentation\/AuthFoundation\/KeychainError\/failureReason","kind":"symbol","type":"topic","url":"\/documentation\/authfoundation\/keychainerror\/failurereason"}}}